src/main/java/frc/io/subsystems/WinchIO.java

```diff
@@ -0,0 +1,127 @@
+package frc.io.subsystems;
+
+// Import required Libraries
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+// Import required Classes
+import frc.robot.Constants;
+import frc.util.pid.SparkMaxConstants;
+import frc.util.pid.SparkMaxPID;
+
+public class WinchIO implements IIO{
```

Comment:
What system is the winch? I don't remember hearing of this

Reply:
I decided it was best to keep climb functionality separate from the arms since they are used by both shooter and climb, climb is just called winch!

Reply:
Sounds good!